### ISSUE TITLE
Difference between kubectl top and kube-capacity Plug-In --util values

### ISSUE THREAD

**Issue by VF-mbrauer** [robscott/kube-capacity#16]
When using the following command: 
```
~$ kubectl top nodes

NAME                                          CPU(cores)   CPU%   MEMORY(bytes)   MEMORY%   
ip-10-1-0-100.eu-central-1.compute.internal   589m         7%     12429Mi         81%     
```
there is a difference in the utilization:
```
~$ kubectl resource-capacity --util --sort cpu.util | sed -r '/^\s*$/d'
NODE                                          CPU REQUESTS    CPU LIMITS       CPU UTIL       MEMORY REQUESTS   MEMORY LIMITS     MEMORY UTIL
ip-10-1-0-100.eu-central-1.compute.internal   6245m (78%)     9400m (117%)     346m (4%)      13177Mi (86%)     18224Mi (119%)    10593Mi (69%)

```
Why is that the case? Is this a bug, or do we need to take anything else into consideration?
Thanks.

**Comment by robscott** [comment]
Hey @VF-mbrauer, that's a great question. It looks like `kubectl top` is using node metrics there (which makes sense), where I'm using the sum of pod metrics to get a node total. The difference seems to be any system utilization happening on a node that isn't represented as part of a pod. I think it would likely make sense to update my logic to also fetch node metrics, instead of summing all the pods like I currently am. I don't have a ton of extra time right now, but will try to fit this in when I do.

**Comment by robscott** [comment]
Hey @VF-mbrauer, I've published a new 0.3.2 release that should fix this. Let me know if you run into any issues with it. Thanks for reporting this!

**Comment by VF-mbrauer** [comment]
Thanks a lot @robscott. Will do and let you know.
